content/workshops/security_containers/exercise1.1.adoc

```diff
@@ -79,15 +80,15 @@ For example, to create a checksum for a Redis release:
 
 [source,bash]
 ----
-curl -s -o redis.tar.gz \
-http://download.redis.io/releases/redis-3.2.8.tar.gz
+curl http://download.redis.io/releases/redis-3.2.8.tar.gz --output redis.tar.gz --progress-bar
 ----
 
 [source,bash]
 ----
 sha1sum -b redis.tar.gz
 
-  6780d1abb66f33a97aad0edbe020403d0a15b67f *redis.tar.gz
+
+6780d1abb66f33a97aad0edbe020403d0a15b67f *redis.tar.gz
```

Comment:
sha1 is probably good enough security.  But in case you do want to update this, the sha256 is ```61b373c23d18e6cc752a69d5ab7f676c6216dc2853e46750a8c4ed791d68482c *redis.tar.gz```